addons/im_livechat/views/discuss_channel_views.xml

```diff
@@ -62,56 +62,32 @@
+                                                            <field class="fw-bolder" name="author_id"/>
+                                                            <field class="fw-bolder" name="author_guest_id"/>
+                                                            Replied to
+                                                            <field class="fw-bolder" name="parent_author_name"/>
+                                                            <field class="bg-200 p-2 rounded" name="parent_body" widget="html"/>
                                                         </div>
-                                                        <div class="d-flex" t-att-class="{'ms-3': isMessageReply}">
-                                                            <t t-if="record.author_avatar.raw_value">
-                                                                <img t-att-src="kanban_image('mail.message', 'author_avatar', record.id.raw_value)" alt="Avatar" class="o_image_64_cover o-livechat-HistoryKanban-authorAvatar rounded"/>
-                                                             </t>
-                                                             <t t-else=""><img alt="Anonymous" src="/mail/static/src/img/smiley/avatar.jpg" class="o_image_64_cover o-livechat-HistoryKanban-authorAvatar rounded"/></t>
-                                                            <div class="ms-2 flex-grow-1">
-                                                                <p class="m-0"><strong>
-                                                                    <t t-if="record.author_id.raw_value"><field name="author_id"/></t>
-                                                                    <t t-else=""><field name="author_guest_id"/></t>
-                                                                </strong></p>
-                                                                <p class="m-0 o-livechat-HistoryKanban-body">
-                                                                    <t t-if="record.body.raw_value"><field name="body" widget="html"/></t>
-                                                                </p>
-                                                            </div>
-                                                            <field t-if="!isMessageReply" class="d-none d-sm-block flex-shrink-0" name="date"/>
-                                                        </div>
-                                                        <field class="align-self-end d-block d-sm-none" name="date"/>
+                                                        <field class="d-none d-sm-block flex-shrink-0" name="date"/>
+                                                    </div>
+                                                    <div class="d-flex">
+                                                        <aside class="o_kanban_aside_small">
```

Comment:
using aside/main not as root of the card won't apply the rules enforced by the framework. You should probably go with divs here to avoid confusion